Project.toml

```
@@ -5,17 +5,22 @@ version = "1.14.0"
 [deps]
 ConstructionBase = "187b0558-2788-49d3-abe0-74a17ed4e7c9"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+InverseFunctions = "3587e190-3f89-42d0-90ee-14403ec27112"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
+[weakdeps]
+InverseFunctions = "3587e190-3f89-42d0-90ee-14403ec27112"
+
+[extensions]
+InverseFunctionsUnitfulExt = "InverseFunctions"
+
 [compat]
 Aqua = "0.6"
```

```suggestion
Aqua = "0.6.3"
```
see my other comment